linode/manager
# fix: [M3-7997] - Charts Y-axis values are trimmed when scale is increased

## Description 📝
Fix chart y-axis values being trimmed for large values and add support for Billion and Trillion values

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/linode/manager/assets/115299789/828b12f0-651f-46fc-9a9a-4821e4c24333) | ![image](https://github.com/linode/manager/assets/115299789/b641c9a2-1eaa-40ed-9ed1-d2fb21c3b684) |
| ![Screenshot 2024-04-16 at 2 41 39 PM](https://github.com/linode/manager/assets/115299789/64cb9217-15a3-4c22-9b28-821fccd6d241) | ![image](https://github.com/linode/manager/assets/115299789/3fa7b2ee-a806-469e-94d1-51325321495f) |

## How to test 🧪

### Verification steps
- Go to a Longview client details page and check the charts (you may need to refresh/play with the dropdown if there's an error loading the data). The y-axis label values should no longer be cut off

## As an Author I have considered 🤔

*Check all that apply*
